site/_resources/upgrading.md

```diff
@@ -9,21 +9,66 @@ toc: h1,h2
 
 This document describes the changes needed to upgrade your Contour installation.
 
-# Upgrading Contour 1.0.1 to 1.1.0
+# Upgrading Contour 1.1.0 to 1.2.0
 
-Contour 1.1.0 is the current stable release.
+Contour 1.2.0 is the current stable release.
 
 <div class="alert-deprecation">
-<h3><b>Deprecation Notice</b></h3>
-<p>The <code>IngressRoute</code> CRD has been deprecated and will not receive further updates. Contour 1.1.0 continues to support the IngressRoute API, however we anticipate it will be removed in the future.</p>
-<p>Please see the documentation for <a href="{% link docs/v1.1.0/httpproxy.md %}"><code>HTTPProxy</code></a>, which is the successor to <code>IngressRoute</code>.
-You can also read the <a href="{% link _guides/ingressroute-to-httpproxy.md %}">IngressRoute to HTTPProxy upgrade</a> guide.</p>
+<h3>Deprecation Notice</h3>
+<p>The <code>IngressRoute</code> CRD has been deprecated and will not receive further updates. Contour 1.2.0 continues to support the IngressRoute API, however we anticipate it will be removed in the future.</p>
+<p>Please see the documentation for <a href="{% link docs/{{site.latest}}/httpproxy.md %}"><code>HTTPProxy</code></a>, which is the successor to <code>IngressRoute</code>. You can also read the <a href="{% link _guides/ingressroute-to-httpproxy.md %}">IngressRoute to HTTPProxy upgrade</a> guide.</p>
 </div>
 
 &nbsp;
 
 ## Recommended Envoy version
 
+All users should ensure the Envoy image version is `docker.io/envoyproxy/envoy:v1.13.0`.
+
+Please see the [Envoy Release Notes][17] for information about issues fixed in Envoy 1.13.0.
+
+## The easy way to upgrade
+
+If the following are true for you:
+
+ * Your installation is in the `projectcontour` namespace.
+ * You are using one of the [example][1] deployments.
+ * Your cluster can take few minutes of downtime.
+
+Then the simplest way to upgrade to 1.2.0 is to delete the `projectcontour` namespace and reapply one of the example configurations.
+From the root directory of the repository:
+
+```
+$ kubectl delete namespace projectcontour
+$ kubectl apply -f examples/<your-desired-deployment>
```

Comment:
I didn't previously notice that we recommended this. The only thing you can apply here is `examples/contour`, it's likely to be confusing to suggest otherwise.

I think this should advise `https://projectcontour.io/quickstart/contour.yaml`, like we do elsewhere.